### DISCUSSION TITLE
Hoobs support

### DISCUSSION THREAD

**Discussion by iANGELgr** [diyhue/diyHue#607] [Q&A]
Hello, I install DIYHUE at hoobs PI. 
The installation was successful and Philips hue app / Hue essentials recognise the "virtual" hue bringe (DIYHUE) but I can't "push" the button.
I use the url http://IP_address/hue/linkbutton but shows a blank page inside the Hoobs interface.

How can I push the button? with code maybe?

**Comment by mariusmotea** [comment]
You need to provide logs to see what error is generated on the hue emulator side

**Reply by iANGELgr** [reply]
@mariusmotea I dont receive any error at log. only the following when I restart the Hoobs

21/04/2021, 14:24:35 HOOBS listening on port 80.
21/04/2021, 14:24:39 Initializing HAP-NodeJS v0.9.3...
21/04/2021, 14:24:40 No plugins installed.
21/04/2021, 14:24:40 Preparing Advertiser for 'HOOBS 23D4' using bonjour-hap backend!
21/04/2021, 14:24:40 Starting to advertise 'HOOBS 23D4' using bonjour-hap backend!
21/04/2021, 14:24:40 Bridge is running on port 51826.

When I start, I went at Hoobs Terminal and run the following code for automatic install of DIYHUE

curl -fsSL https://get.diyhue.org -o install.sh
sudo bash install.sh

After the process, at the end, I receive message of installation successfully , try to search for bridge and after this myHoobs is discoverable at Philips Hue app & Hue Essentials AS NEW BRIDGE! And the apps tells me to press the button.

The problem is, when I run this url

http://{IP_ADDRESS}/hue/linkbutton 

shows me the login interface of Hoobs hub and when I login the page is blank (check the screenshot)
![Screenshot](https://user-images.githubusercontent.com/2945030/115550041-10859380-a2b2-11eb-884a-83acb9678667.png)

The Hoobs hub is Raspberry Pi and you can run other plugins if you want too.

How can I fix the problem please???

**Reply by iANGELgr** [reply]
@mariusmotea do you want to re-install it to see the log again????

**Reply by mariusmotea** [reply]
The bridge emulator logs i need to see how this try to pair. If it use the cloud based detection we have no options.

**Reply by iANGELgr** [reply]
@mariusmotea the only log that I found is this but shows MyHoobs without any plugin

hoobs@hoobs:/home/hoobs/.hoobs/etc $ sudo hoobs service log
Started HOOBS.
HOOBS listening on port 80.
Initializing HAP-NodeJS v0.9.3...
No plugins installed.
Preparing Advertiser for 'HOOBS 23D4' using bonjour-hap backend!
Starting to advertise 'HOOBS 23D4' using bonjour-hap backend!
Bridge is running on port 51826.
   hoobs : TTY=pts/0 ; PWD=/home/hoobs/.hoobs/etc ; USER=root ; COMMAND=/usr/local/bin/hoobs service log
pam_unix(sudo:session): session opened for user root by (uid=0)
pam_unix(sudo:session): session closed for user root
   hoobs : TTY=pts/0 ; PWD=/home/hoobs/.hoobs/etc ; USER=root ; COMMAND=/usr/local/bin/hoobs service log
pam_unix(sudo:session): session opened for user root by (uid=0)
pam_unix(sudo:session): session closed for user root
   hoobs : TTY=pts/0 ; PWD=/home/hoobs/.hoobs/etc ; USER=root ; COMMAND=/usr/local/bin/hoobs service log
pam_unix(sudo:session): session opened for user root by (uid=0)

I run 
Bridge Name | HOOBS
-- | --
HOOBS Version | 3.3.5
Node Version | 12.16.2

and System

Manufacturer | Raspberry Pi Foundation
-- | --
Model | Raspberry Pi 3 Model B Rev 1.2
Version | a52082
Serial | XXXXXXXXXXXXXXXXXXX
Uuid |  
Sku | -
Virtual | false
Raspberry | { "manufacturer": "Stadium", "processor": "BCM2837", "type": "3B", "revision": "1.2" }

Do you have any code to run at Hoobs Terminal?? To find something for DIYHUE?? The virtual bridge is discovered by the apps ;)
The Hoobs system is known system at Raspberry PI.

**Reply by iANGELgr** [reply]
I re-install it from above
LOOK the log please

`hoobs@hoobs:/home/hoobs/.hoobs/etc $ sudo bash install.sh
Please choose a Branch to install
Select Branch by entering the corresponding Number: [Default: Master]  
[1] Master Branch - most stable Release 
[2] Developer Branch - test latest features and fixes - Work in Progress!
Note: Please report any Bugs or Errors with Logs to our GitHub, Discourse or Slack. Thank you!
I go with Nr.: 1
Master selected
 Installing dependencies.
Reading package lists... Done
Building dependency tree       
Reading state information... Done
nmap is already the newest version (7.70+dfsg1-6+deb10u1).
python3 is already the newest version (3.7.3-1).
python3-requests is already the newest version (2.21.0-1).
python3-setuptools is already the newest version (40.8.0-1).
unzip is already the newest version (6.0-23+deb10u2).
0 upgraded, 0 newly installed, 0 to remove and 125 not upgraded.
 Installing Python Astral.
running install
running bdist_egg
running egg_info
creating src/astral.egg-info
writing src/astral.egg-info/PKG-INFO
writing dependency_links to src/astral.egg-info/dependency_links.txt
writing requirements to src/astral.egg-info/requires.txt
writing top-level names to src/astral.egg-info/top_level.txt
writing manifest file 'src/astral.egg-info/SOURCES.txt'
reading manifest file 'src/astral.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no previously-included files matching '*.pyc' found anywhere in distribution
writing manifest file 'src/astral.egg-info/SOURCES.txt'
installing library code to build/bdist.linux-armv7l/egg
running install_lib
running build_py
creating build
creating build/lib
creating build/lib/astral
copying src/astral/__main__.py -> build/lib/astral
copying src/astral/moon.py -> build/lib/astral
copying src/astral/__init__.py -> build/lib/astral
copying src/astral/geocoder.py -> build/lib/astral
copying src/astral/sun.py -> build/lib/astral
copying src/astral/location.py -> build/lib/astral
creating build/bdist.linux-armv7l
creating build/bdist.linux-armv7l/egg
creating build/bdist.linux-armv7l/egg/astral
copying build/lib/astral/__main__.py -> build/bdist.linux-armv7l/egg/astral
copying build/lib/astral/moon.py -> build/bdist.linux-armv7l/egg/astral
copying build/lib/astral/__init__.py -> build/bdist.linux-armv7l/egg/astral
copying build/lib/astral/geocoder.py -> build/bdist.linux-armv7l/egg/astral
copying build/lib/astral/sun.py -> build/bdist.linux-armv7l/egg/astral
copying build/lib/astral/location.py -> build/bdist.linux-armv7l/egg/astral
byte-compiling build/bdist.linux-armv7l/egg/astral/__main__.py to __main__.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/astral/moon.py to moon.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/astral/__init__.py to __init__.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/astral/geocoder.py to geocoder.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/astral/sun.py to sun.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/astral/location.py to location.cpython-37.pyc
creating build/bdist.linux-armv7l/egg/EGG-INFO
copying src/astral.egg-info/PKG-INFO -> build/bdist.linux-armv7l/egg/EGG-INFO
copying src/astral.egg-info/SOURCES.txt -> build/bdist.linux-armv7l/egg/EGG-INFO
copying src/astral.egg-info/dependency_links.txt -> build/bdist.linux-armv7l/egg/EGG-INFO
copying src/astral.egg-info/requires.txt -> build/bdist.linux-armv7l/egg/EGG-INFO
copying src/astral.egg-info/top_level.txt -> build/bdist.linux-armv7l/egg/EGG-INFO
copying src/astral.egg-info/zip-safe -> build/bdist.linux-armv7l/egg/EGG-INFO
creating dist
creating 'dist/astral-2.2-py3.7.egg' and adding 'build/bdist.linux-armv7l/egg' to it
removing 'build/bdist.linux-armv7l/egg' (and everything under it)
Processing astral-2.2-py3.7.egg
Removing /usr/local/lib/python3.7/dist-packages/astral-2.2-py3.7.egg
Copying astral-2.2-py3.7.egg to /usr/local/lib/python3.7/dist-packages
astral 2.2 is already the active version in easy-install.pth

Installed /usr/local/lib/python3.7/dist-packages/astral-2.2-py3.7.egg
Processing dependencies for astral==2.2
Searching for pytz==2021.1
Best match: pytz 2021.1
Processing pytz-2021.1-py3.7.egg
pytz 2021.1 is already the active version in easy-install.pth

Using /usr/local/lib/python3.7/dist-packages/pytz-2021.1-py3.7.egg
Finished processing dependencies for astral==2.2
 Installing Python MQTT.
paho-mqtt-1.5.0/
paho-mqtt-1.5.0/setup.cfg
paho-mqtt-1.5.0/edl-v10
paho-mqtt-1.5.0/notice.html
paho-mqtt-1.5.0/MANIFEST.in
paho-mqtt-1.5.0/src/
paho-mqtt-1.5.0/src/paho/
paho-mqtt-1.5.0/src/paho/mqtt/
paho-mqtt-1.5.0/src/paho/mqtt/__init__.py
paho-mqtt-1.5.0/src/paho/mqtt/client.py
paho-mqtt-1.5.0/src/paho/mqtt/subscribeoptions.py
paho-mqtt-1.5.0/src/paho/mqtt/subscribe.py
paho-mqtt-1.5.0/src/paho/mqtt/reasoncodes.py
paho-mqtt-1.5.0/src/paho/mqtt/matcher.py
paho-mqtt-1.5.0/src/paho/mqtt/packettypes.py
paho-mqtt-1.5.0/src/paho/mqtt/publish.py
paho-mqtt-1.5.0/src/paho/mqtt/properties.py
paho-mqtt-1.5.0/src/paho/__init__.py
paho-mqtt-1.5.0/src/paho_mqtt.egg-info/
paho-mqtt-1.5.0/src/paho_mqtt.egg-info/dependency_links.txt
paho-mqtt-1.5.0/src/paho_mqtt.egg-info/SOURCES.txt
paho-mqtt-1.5.0/src/paho_mqtt.egg-info/requires.txt
paho-mqtt-1.5.0/src/paho_mqtt.egg-info/top_level.txt
paho-mqtt-1.5.0/src/paho_mqtt.egg-info/not-zip-safe
paho-mqtt-1.5.0/src/paho_mqtt.egg-info/PKG-INFO
paho-mqtt-1.5.0/examples/
paho-mqtt-1.5.0/examples/subscribe_callback.py
paho-mqtt-1.5.0/examples/loop_select.py
paho-mqtt-1.5.0/examples/client_sub-class.py
paho-mqtt-1.5.0/examples/client_sub_opts.py
paho-mqtt-1.5.0/examples/client_sub-ws.py
paho-mqtt-1.5.0/examples/publish_multiple.py
paho-mqtt-1.5.0/examples/subscribe_simple.py
paho-mqtt-1.5.0/examples/publish_utf8-27.py
paho-mqtt-1.5.0/examples/client_mqtt_clear_retain.py
paho-mqtt-1.5.0/examples/client_sub-multiple-callback.py
paho-mqtt-1.5.0/examples/client_sub-srv.py
paho-mqtt-1.5.0/examples/loop_asyncio.py
paho-mqtt-1.5.0/examples/client_pub-wait.py
paho-mqtt-1.5.0/examples/client_logger.py
paho-mqtt-1.5.0/examples/aws_iot.py
paho-mqtt-1.5.0/examples/client_session_present.py
paho-mqtt-1.5.0/examples/client_sub.py
paho-mqtt-1.5.0/examples/publish_single.py
paho-mqtt-1.5.0/examples/context.py
paho-mqtt-1.5.0/examples/client_pub_opts.py
paho-mqtt-1.5.0/epl-v10
paho-mqtt-1.5.0/CONTRIBUTING.md
paho-mqtt-1.5.0/setup.py
paho-mqtt-1.5.0/LICENSE.txt
paho-mqtt-1.5.0/PKG-INFO
paho-mqtt-1.5.0/README.rst
paho-mqtt-1.5.0/about.html
running install
running bdist_egg
running egg_info
writing src/paho_mqtt.egg-info/PKG-INFO
writing dependency_links to src/paho_mqtt.egg-info/dependency_links.txt
writing requirements to src/paho_mqtt.egg-info/requires.txt
writing top-level names to src/paho_mqtt.egg-info/top_level.txt
reading manifest file 'src/paho_mqtt.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'src/paho_mqtt.egg-info/SOURCES.txt'
installing library code to build/bdist.linux-armv7l/egg
running install_lib
running build_py
creating build
creating build/lib
creating build/lib/paho
copying src/paho/__init__.py -> build/lib/paho
creating build/lib/paho/mqtt
copying src/paho/mqtt/subscribe.py -> build/lib/paho/mqtt
copying src/paho/mqtt/client.py -> build/lib/paho/mqtt
copying src/paho/mqtt/__init__.py -> build/lib/paho/mqtt
copying src/paho/mqtt/matcher.py -> build/lib/paho/mqtt
copying src/paho/mqtt/subscribeoptions.py -> build/lib/paho/mqtt
copying src/paho/mqtt/publish.py -> build/lib/paho/mqtt
copying src/paho/mqtt/properties.py -> build/lib/paho/mqtt
copying src/paho/mqtt/packettypes.py -> build/lib/paho/mqtt
copying src/paho/mqtt/reasoncodes.py -> build/lib/paho/mqtt
creating build/bdist.linux-armv7l
creating build/bdist.linux-armv7l/egg
creating build/bdist.linux-armv7l/egg/paho
copying build/lib/paho/__init__.py -> build/bdist.linux-armv7l/egg/paho
creating build/bdist.linux-armv7l/egg/paho/mqtt
copying build/lib/paho/mqtt/subscribe.py -> build/bdist.linux-armv7l/egg/paho/mqtt
copying build/lib/paho/mqtt/client.py -> build/bdist.linux-armv7l/egg/paho/mqtt
copying build/lib/paho/mqtt/__init__.py -> build/bdist.linux-armv7l/egg/paho/mqtt
copying build/lib/paho/mqtt/matcher.py -> build/bdist.linux-armv7l/egg/paho/mqtt
copying build/lib/paho/mqtt/subscribeoptions.py -> build/bdist.linux-armv7l/egg/paho/mqtt
copying build/lib/paho/mqtt/publish.py -> build/bdist.linux-armv7l/egg/paho/mqtt
copying build/lib/paho/mqtt/properties.py -> build/bdist.linux-armv7l/egg/paho/mqtt
copying build/lib/paho/mqtt/packettypes.py -> build/bdist.linux-armv7l/egg/paho/mqtt
copying build/lib/paho/mqtt/reasoncodes.py -> build/bdist.linux-armv7l/egg/paho/mqtt
byte-compiling build/bdist.linux-armv7l/egg/paho/__init__.py to __init__.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/paho/mqtt/subscribe.py to subscribe.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/paho/mqtt/client.py to client.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/paho/mqtt/__init__.py to __init__.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/paho/mqtt/matcher.py to matcher.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/paho/mqtt/subscribeoptions.py to subscribeoptions.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/paho/mqtt/publish.py to publish.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/paho/mqtt/properties.py to properties.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/paho/mqtt/packettypes.py to packettypes.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/paho/mqtt/reasoncodes.py to reasoncodes.cpython-37.pyc
creating build/bdist.linux-armv7l/egg/EGG-INFO
copying src/paho_mqtt.egg-info/PKG-INFO -> build/bdist.linux-armv7l/egg/EGG-INFO
copying src/paho_mqtt.egg-info/SOURCES.txt -> build/bdist.linux-armv7l/egg/EGG-INFO
copying src/paho_mqtt.egg-info/dependency_links.txt -> build/bdist.linux-armv7l/egg/EGG-INFO
copying src/paho_mqtt.egg-info/not-zip-safe -> build/bdist.linux-armv7l/egg/EGG-INFO
copying src/paho_mqtt.egg-info/requires.txt -> build/bdist.linux-armv7l/egg/EGG-INFO
copying src/paho_mqtt.egg-info/top_level.txt -> build/bdist.linux-armv7l/egg/EGG-INFO
creating dist
creating 'dist/paho_mqtt-1.5.0-py3.7.egg' and adding 'build/bdist.linux-armv7l/egg' to it
removing 'build/bdist.linux-armv7l/egg' (and everything under it)
Processing paho_mqtt-1.5.0-py3.7.egg
removing '/usr/local/lib/python3.7/dist-packages/paho_mqtt-1.5.0-py3.7.egg' (and everything under it)
creating /usr/local/lib/python3.7/dist-packages/paho_mqtt-1.5.0-py3.7.egg
Extracting paho_mqtt-1.5.0-py3.7.egg to /usr/local/lib/python3.7/dist-packages
paho-mqtt 1.5.0 is already the active version in easy-install.pth

Installed /usr/local/lib/python3.7/dist-packages/paho_mqtt-1.5.0-py3.7.egg
Processing dependencies for paho-mqtt==1.5.0
Finished processing dependencies for paho-mqtt==1.5.0
 Installing WebSocket for Python.
running install
running bdist_egg
running egg_info
creating ws4py.egg-info
writing ws4py.egg-info/PKG-INFO
writing dependency_links to ws4py.egg-info/dependency_links.txt
writing top-level names to ws4py.egg-info/top_level.txt
writing manifest file 'ws4py.egg-info/SOURCES.txt'
reading manifest file 'ws4py.egg-info/SOURCES.txt'
writing manifest file 'ws4py.egg-info/SOURCES.txt'
installing library code to build/bdist.linux-armv7l/egg
running install_lib
running build_py
creating build
creating build/lib
creating build/lib/ws4py
copying ws4py/messaging.py -> build/lib/ws4py
copying ws4py/compat.py -> build/lib/ws4py
copying ws4py/manager.py -> build/lib/ws4py
copying ws4py/__init__.py -> build/lib/ws4py
copying ws4py/exc.py -> build/lib/ws4py
copying ws4py/async_websocket.py -> build/lib/ws4py
copying ws4py/utf8validator.py -> build/lib/ws4py
copying ws4py/streaming.py -> build/lib/ws4py
copying ws4py/framing.py -> build/lib/ws4py
copying ws4py/websocket.py -> build/lib/ws4py
creating build/lib/ws4py/client
copying ws4py/client/geventclient.py -> build/lib/ws4py/client
copying ws4py/client/__init__.py -> build/lib/ws4py/client
copying ws4py/client/threadedclient.py -> build/lib/ws4py/client
copying ws4py/client/tornadoclient.py -> build/lib/ws4py/client
creating build/lib/ws4py/server
copying ws4py/server/tulipserver.py -> build/lib/ws4py/server
copying ws4py/server/__init__.py -> build/lib/ws4py/server
copying ws4py/server/cherrypyserver.py -> build/lib/ws4py/server
copying ws4py/server/wsgirefserver.py -> build/lib/ws4py/server
copying ws4py/server/geventserver.py -> build/lib/ws4py/server
copying ws4py/server/wsgiutils.py -> build/lib/ws4py/server
creating build/bdist.linux-armv7l
creating build/bdist.linux-armv7l/egg
creating build/bdist.linux-armv7l/egg/ws4py
copying build/lib/ws4py/messaging.py -> build/bdist.linux-armv7l/egg/ws4py
copying build/lib/ws4py/compat.py -> build/bdist.linux-armv7l/egg/ws4py
copying build/lib/ws4py/manager.py -> build/bdist.linux-armv7l/egg/ws4py
copying build/lib/ws4py/__init__.py -> build/bdist.linux-armv7l/egg/ws4py
copying build/lib/ws4py/exc.py -> build/bdist.linux-armv7l/egg/ws4py
copying build/lib/ws4py/async_websocket.py -> build/bdist.linux-armv7l/egg/ws4py
copying build/lib/ws4py/utf8validator.py -> build/bdist.linux-armv7l/egg/ws4py
creating build/bdist.linux-armv7l/egg/ws4py/client
copying build/lib/ws4py/client/geventclient.py -> build/bdist.linux-armv7l/egg/ws4py/client
copying build/lib/ws4py/client/__init__.py -> build/bdist.linux-armv7l/egg/ws4py/client
copying build/lib/ws4py/client/threadedclient.py -> build/bdist.linux-armv7l/egg/ws4py/client
copying build/lib/ws4py/client/tornadoclient.py -> build/bdist.linux-armv7l/egg/ws4py/client
copying build/lib/ws4py/streaming.py -> build/bdist.linux-armv7l/egg/ws4py
copying build/lib/ws4py/framing.py -> build/bdist.linux-armv7l/egg/ws4py
copying build/lib/ws4py/websocket.py -> build/bdist.linux-armv7l/egg/ws4py
creating build/bdist.linux-armv7l/egg/ws4py/server
copying build/lib/ws4py/server/tulipserver.py -> build/bdist.linux-armv7l/egg/ws4py/server
copying build/lib/ws4py/server/__init__.py -> build/bdist.linux-armv7l/egg/ws4py/server
copying build/lib/ws4py/server/cherrypyserver.py -> build/bdist.linux-armv7l/egg/ws4py/server
copying build/lib/ws4py/server/wsgirefserver.py -> build/bdist.linux-armv7l/egg/ws4py/server
copying build/lib/ws4py/server/geventserver.py -> build/bdist.linux-armv7l/egg/ws4py/server
copying build/lib/ws4py/server/wsgiutils.py -> build/bdist.linux-armv7l/egg/ws4py/server
byte-compiling build/bdist.linux-armv7l/egg/ws4py/messaging.py to messaging.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/ws4py/compat.py to compat.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/ws4py/manager.py to manager.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/ws4py/__init__.py to __init__.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/ws4py/exc.py to exc.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/ws4py/async_websocket.py to async_websocket.cpython-37.pyc
  File "build/bdist.linux-armv7l/egg/ws4py/async_websocket.py", line 87
    asyncio.async(closeit())
                ^
SyntaxError: invalid syntax

byte-compiling build/bdist.linux-armv7l/egg/ws4py/utf8validator.py to utf8validator.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/ws4py/client/geventclient.py to geventclient.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/ws4py/client/__init__.py to __init__.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/ws4py/client/threadedclient.py to threadedclient.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/ws4py/client/tornadoclient.py to tornadoclient.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/ws4py/streaming.py to streaming.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/ws4py/framing.py to framing.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/ws4py/websocket.py to websocket.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/ws4py/server/tulipserver.py to tulipserver.cpython-37.pyc
  File "build/bdist.linux-armv7l/egg/ws4py/server/tulipserver.py", line 43
    f = asyncio.async(self.handle_initial_handshake())
                    ^
SyntaxError: invalid syntax

byte-compiling build/bdist.linux-armv7l/egg/ws4py/server/__init__.py to __init__.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/ws4py/server/cherrypyserver.py to cherrypyserver.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/ws4py/server/wsgirefserver.py to wsgirefserver.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/ws4py/server/geventserver.py to geventserver.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/ws4py/server/wsgiutils.py to wsgiutils.cpython-37.pyc
creating build/bdist.linux-armv7l/egg/EGG-INFO
copying ws4py.egg-info/PKG-INFO -> build/bdist.linux-armv7l/egg/EGG-INFO
copying ws4py.egg-info/SOURCES.txt -> build/bdist.linux-armv7l/egg/EGG-INFO
copying ws4py.egg-info/dependency_links.txt -> build/bdist.linux-armv7l/egg/EGG-INFO
copying ws4py.egg-info/top_level.txt -> build/bdist.linux-armv7l/egg/EGG-INFO
zip_safe flag not set; analyzing archive contents...
creating dist
creating 'dist/ws4py-0.3.4-py3.7.egg' and adding 'build/bdist.linux-armv7l/egg' to it
removing 'build/bdist.linux-armv7l/egg' (and everything under it)
Processing ws4py-0.3.4-py3.7.egg
Removing /usr/local/lib/python3.7/dist-packages/ws4py-0.3.4-py3.7.egg
Copying ws4py-0.3.4-py3.7.egg to /usr/local/lib/python3.7/dist-packages
ws4py 0.3.4 is already the active version in easy-install.pth

Installed /usr/local/lib/python3.7/dist-packages/ws4py-0.3.4-py3.7.egg
Processing dependencies for ws4py==0.3.4
Finished processing dependencies for ws4py==0.3.4
 Installing zeroconf for Python.
running install
running bdist_egg
running egg_info
creating zeroconf.egg-info
writing zeroconf.egg-info/PKG-INFO
writing dependency_links to zeroconf.egg-info/dependency_links.txt
writing requirements to zeroconf.egg-info/requires.txt
writing top-level names to zeroconf.egg-info/top_level.txt
writing manifest file 'zeroconf.egg-info/SOURCES.txt'
reading manifest file 'zeroconf.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'zeroconf.egg-info/SOURCES.txt'
installing library code to build/bdist.linux-armv7l/egg
running install_lib
running build_py
creating build
creating build/lib
creating build/lib/zeroconf
copying zeroconf/__init__.py -> build/lib/zeroconf
copying zeroconf/test.py -> build/lib/zeroconf
copying zeroconf/py.typed -> build/lib/zeroconf
creating build/bdist.linux-armv7l
creating build/bdist.linux-armv7l/egg
creating build/bdist.linux-armv7l/egg/zeroconf
copying build/lib/zeroconf/__init__.py -> build/bdist.linux-armv7l/egg/zeroconf
copying build/lib/zeroconf/py.typed -> build/bdist.linux-armv7l/egg/zeroconf
copying build/lib/zeroconf/test.py -> build/bdist.linux-armv7l/egg/zeroconf
byte-compiling build/bdist.linux-armv7l/egg/zeroconf/__init__.py to __init__.cpython-37.pyc
byte-compiling build/bdist.linux-armv7l/egg/zeroconf/test.py to test.cpython-37.pyc
creating build/bdist.linux-armv7l/egg/EGG-INFO
copying zeroconf.egg-info/PKG-INFO -> build/bdist.linux-armv7l/egg/EGG-INFO
copying zeroconf.egg-info/SOURCES.txt -> build/bdist.linux-armv7l/egg/EGG-INFO
copying zeroconf.egg-info/dependency_links.txt -> build/bdist.linux-armv7l/egg/EGG-INFO
copying zeroconf.egg-info/not-zip-safe -> build/bdist.linux-armv7l/egg/EGG-INFO
copying zeroconf.egg-info/requires.txt -> build/bdist.linux-armv7l/egg/EGG-INFO
copying zeroconf.egg-info/top_level.txt -> build/bdist.linux-armv7l/egg/EGG-INFO
creating dist
creating 'dist/zeroconf-0.28.6-py3.7.egg' and adding 'build/bdist.linux-armv7l/egg' to it
removing 'build/bdist.linux-armv7l/egg' (and everything under it)
Processing zeroconf-0.28.6-py3.7.egg
removing '/usr/local/lib/python3.7/dist-packages/zeroconf-0.28.6-py3.7.egg' (and everything under it)
creating /usr/local/lib/python3.7/dist-packages/zeroconf-0.28.6-py3.7.egg
Extracting zeroconf-0.28.6-py3.7.egg to /usr/local/lib/python3.7/dist-packages
zeroconf 0.28.6 is already the active version in easy-install.pth

Installed /usr/local/lib/python3.7/dist-packages/zeroconf-0.28.6-py3.7.egg
Processing dependencies for zeroconf==0.28.6
Searching for ifaddr==0.1.7
Best match: ifaddr 0.1.7
Processing ifaddr-0.1.7-py3.7.egg
ifaddr 0.1.7 is already the active version in easy-install.pth

Using /usr/local/lib/python3.7/dist-packages/ifaddr-0.1.7-py3.7.egg
Finished processing dependencies for zeroconf==0.28.6
 Installing Hue Emulator.
 Existing installation found, performing upgrade.
 Installation completed. Open Hue app and search for bridges.`

**Reply by iANGELgr** [reply]
After I go to this URL (Virtual Philips Hue bridge / DIYHUE)

http://10.0.4.22/hue/linkbutton

And I saw HOOBS login page and not DIYHUE login
so I put HOOBS login details and after the page is blank! :(
